### DISCUSSION TITLE
currently playing section in dashboard (F9) tab shows offline players

### DISCUSSION THREAD

**Discussion by Gamingzone123** [ppy/osu#16395] [Q&A] [answered]
not sure what the issue is but there certainly is one

![image](https://user-images.githubusercontent.com/47021866/148734700-c949afc2-35e0-4522-8264-2d8234bcb567.png)

edit: I've been told that its due to Lazer not updating players status as online, still an issue though, if not already in the issues tab (i searched for the currently playing section displaying offline players, not Lazer updating online status)

**Comment by peppy** [comment, marked as answer]
Online status isn't currently being sent/supported anywhere. It probably needs to be at some point. We still need to figure what it means, too (ie. how much of online systems will be disabled when it's set to "offline".

**Reply by Gamingzone123** [reply]
gotcha, might want to make an issue or mark the discurrion to ensure its not forgotten though

**Reply by AuracleTech** [reply]
I consider as long as the TCP connection to the IRC is live the player should be considered online
